mental_health/index.html

```diff
@@ -21,7 +21,7 @@ <h3>Your instructors</h3>
 
     <h3>Financial stress</h3>
     <p>
-      For financial concerns, you can reach out directly to Darren Smith on
+      For financial concerns, you can reach out directly to Tamika Shipp on
       Slack; he can connect you with various services (especially in the
```

Comment:
Not sure what's going on with these files from my and scott's recent updates...

Reply:
I believe the latest changes to `gh-pages` are being added in here b/c of Travis branching off `gh-pages` after `development` was created. Not a bad idea to sync `development` and `gh-pages` every once in a while to keep them in sync w/ each other.

Reply:
Thanks for catching this Will and Nick!  I'll make sure to branch off from the `development` branch in the future!